mono/dis/util.c

```diff
@@ -20,9 +20,8 @@
 #endif
 
 /**
- * map:
- * @code: code to lookup in table
- * @table: table to decode code
+ * \param code code to lookup in table
```

Comment:
Ah, there is http://docs.go-mono.com/?link=root:/embed linked from http://www.mono-project.com/docs/advanced/embedding/ but I’m not sure how it’s generated.